libraries/provision/ansible/playbooks/install-couchbase-server-package.yml

```diff
@@ -128,39 +127,73 @@
       shell: "{{ couchbase_server_home_path }}/bin/couchbase-cli node-init -c {{ couchbase_server_node }}:{{ couchbase_server_admin_port }} --user={{ couchbase_server_admin }} --password={{ couchbase_server_password }} --node-init-hostname={{ couchbase_server_node }}"
       when: "'3.0' in couchbase_server_package_name or '3.1' in couchbase_server_package_name"
 
-    - name: COUCHBASE SERVER | Initialize primary node | configure IPv4
+    - name: COUCHBASE SERVER | Initialize primary node (4.X and 5.X) | configure IPv4
+      shell: "{{ couchbase_server_home_path }}/bin/couchbase-cli node-init -c {{ couchbase_server_node }}:{{ couchbase_server_admin_port }} --user={{ couchbase_server_admin }} --password={{ couchbase_server_password }} --node-init-hostname={{ couchbase_server_node }}"
+      when: "{{ cb_major_version['stdout'] | search('[4-5]')}} and not ipv6_enabled"
```

Comment:
can you verify that major version is 15 or 14, it still skips and this condition applies only for 4. or 5. , but not for 15. or 14.

Reply:
tuned regex pattern to check single digit number if version is 4.x or 5.x and check single digit + not between 1 to 5 if version is 6.x or above

Reply:
done